backend/app/imageconversion.py

```diff
@@ -0,0 +1,62 @@
+import os
+import subprocess
+from tqdm import tqdm
+import sys
+
+
+def convert(old_path, new_path, old_filetype, new_filetype, commands=None):
+    """
+
+    :param old_path: (str) path to directory containing images
+            [ex. 'Users/bob/Desktop/']
+    :param new_path: (str) path where output is intended to be saved in
+            [ex. 'Users/bob/Desktop/new/']
+    :param old_filetype: (str) image format of input images
+            [ex. 'TIF']
+    :param new_filetype: (str) desired output image format
+            [ex. 'jpg']
+    :param commands: (list) commands to apply to images, default = None
+            [ex. ['-quality, '20%']
+    :return: None, output produced in same directory
+    """
+
+    # creating new directory for output
+    new_dir = new_path
+    try:
+        os.mkdir(new_dir)
+    except FileExistsError:
+        pass
+
+    # str of commands to apply to images
+    command_str = ''
+    print(commands)
```

Comment:
Let's remove one of these print statements. Maybe adding a string would help, as well, like: "List of commands running on each image"